Oqtane.Client/Extensions/OqtaneLocalizationExtensions.cs

```diff
@@ -0,0 +1,22 @@
+namespace Microsoft.Extensions.Localization
+{
+    public static class OqtaneLocalizationExtensions
+    {
+        /// <summary>
+        /// Gets the string resource for the specified key and returns the value if the resource does not exist
+        /// </summary>
+        /// <param name="localizer"></param>
+        /// <param name="key">the static key used to identify the string resource</param>
+        /// <param name="value">the default value if the resource for the static key does not exist</param>
+        /// <returns></returns>
+        public static string GetString(this IStringLocalizer localizer, string key, string value)
```

Comment:
@sbwalker the localization APIs is already return the missing resource, in case you need to set a default value what the scenario or use case that this method can be used for?

Reply:
@hishamco the problem is documented here https://github.com/oqtane/oqtane.translations/issues/34 and was earlier raised here https://github.com/oqtane/oqtane.translations/issues/17:

3. Localization implemented within components will use static keys. This change is being made to ensure that language specific translations are not broken when static text messages are modified in UI components. The standard method for .NET Core localization was theoretically optimized to make software development easier; however it is a nightmare for translators because the key values are not static and often change - breaking the key lookup linkage. 

For example, logic such as Localizer["Please Enter All Fields And Ensure Passwords Match And Are Greater Than 5 Characters In Length"] may require a change to the message - which would break the lookup for all existing resource files for other languages. This will be changed to use an extension method Localizer.GetString("Message.PasswordValidation", "Please Enter All Fields And Ensure Passwords Match And Are Greater Than 5 Characters In Length") where "Message.PasswordValidation" is a static key that will never change and therefore can be relied upon by translators.